docs/ref/models/querysets.txt

```diff
@@ -801,6 +801,48 @@ typically caches its results. If the data in the database might have changed
 since a ``QuerySet`` was evaluated, you can get updated results for the same
 query by calling ``all()`` on a previously evaluated ``QuerySet``.
 
+``union()``
+~~~~~~~~~~~
+
+.. method:: union(*other_qs, all=False)
+
+.. versionadded:: 1.11
+
+Uses SQL's ``UNION`` operator to combine the results of two or more
+``QuerySet``\s. For example:
+
+    >>> qs1.union(qs2, qs3)
+
+Each ``QuerySet`` must have the same number of columns and the columns
+must also have similar data types. (same model?)
+
+The ``UNION`` operator selects only distinct values by default. To allow
+duplicate values, use the ``all=True`` argument.
+
+``intersect()``
```

Comment:
I renamed that to `intersection` in my last update to be consistent with python sets, does that sound sensible, or should I go back to `intersect`